README.md

```diff
@@ -1,4 +1,42 @@
 <p align="center"><img src="https://avatars1.githubusercontent.com/u/58499608?s=100&v=4" alt="One Platform Logo"></p>
 
 # One Platform
-An integrated application hosting platform.
+![Version](https://img.shields.io/badge/version-0.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+[![License: MIT](https://img.shields.io/github/license/1-Platform/one-platform)](https://github.com/1-Platform/one-platform/blob/master/LICENSE)
+
+An integrated application hosting platform which allows you to host your SPAs. It also provides common services like
+- Authentication and Authorization
+- Notifications Framework
+- Feedback Framework
+
+
+### 🏠 [Homepage](https://github.com/1-Platform/one-platform#readme)
+
+## Install
+
+```bash
+npm install
+```
+
```

Comment:
Can you add lerna bootstrap command here (https://lerna.js.org/#command-bootstrap)

Reply:
got it